### DISCUSSION TITLE
Dataset partial word matching is not working

### DISCUSSION THREAD

**Discussion by AlbSed** [ckan/ckan#7416] [General]
## Bug
When you try to search a dataset only from part of the word it doesn't find anything.

<img width="960" alt="ckan_2" src="https://user-images.githubusercontent.com/106113795/220568170-d1100d0a-7a28-42eb-89c7-8368f007985c.PNG">

Only from complete word it is searchable:

<img width="960" alt="ckan_1" src="https://user-images.githubusercontent.com/106113795/220568492-7d8a29c1-cd16-4739-9b20-a952ea9ae64b.PNG">

### Additional details
I'm trying to create a plugin for CKAN which solves this kind of issue, but it doesn't seems to be working.
I tried 2 different solutions but still I wasn't able to do so.

1. I tried to create updated version of 'schema.xml' file with custom fields, cofigurate CKAN to use it and rebuild the search-index. The plugin is working by the logs but in the CKAN it doesn't changed.

2. I tried to update the 'schema.xml' file within the plugin using the 'IDatasetForm' plugin interface, but again it is not working...

Could someone help me with this please?

**Comment by pdelboca** [comment]
Hello @AlbSed !

This is currently not a but but a "feature". Is how search works in CKAN since it relies in Solr to perform it.

You can look at the docs to understand how search works in detail: https://docs.ckan.org/en/latest/user-guide.html#search-in-detail. The type of search you are looking for can be achieved with: `title:Pon*`.
